components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -237,6 +237,23 @@ be lost if SCREAM_HACK_XML is not enabled.
     <!-- CLD fraction -->
     <cldFraction inherit="atm_proc_base"/>
 
+    <!-- MAM4xx namelist options -->
+    <!-- MAM4xx-Optics -->
+    <mam4_optics inherit="atm_proc_base">
+      <mam4_mode1_physical_properties_file type="file">${DIN_LOC_ROOT}/atm/scream/mam4xx/physprops/mam4_mode1_rrtmg_aeronetdust_c20240206.nc</mam4_mode1_physical_properties_file>
```

Comment:
Is the only reason to copy all of these from rrtmg the issue with naming of the field tags? If so, I don't think the rrtmgp interface has inherent dependence on those names, and it might be easier to just change the tag names. Alternatively, there is an issue between rrtmg and rrtmgp where the band ordering for the sw bands differs between the two codes; in EAM, we decided to leave all these files untouched, read them in with the wrong band ordering, and then re-order them once read in. This allowed us to use the same files for both RRTMG and RRTMGP. But, if we have to create new files here anyways, maybe it would be worthwhile to fix this so we do not need to add this reordering to the eamxx impl.

Reply:
That's a good point. Field tags are one of the reasons. The other reason is that SCREAM needs a time dimension added to the files. Therefore, we had to create these new files.

It will be a lot cleaner if we avoid the band reordering like you mentioned. I will create a GitHub issue and fix it in a subsequent PR.

Reply:
@brhillman if we change the name of the field tags (e.g,., `lw_band ` to`lwband` ), NetCDF files with `lw_band ` field tag will need to be changed. I am assuming  rrtmgp or other process relies of files with dimension name `lw_band`/`sw_band`.

Reply:
I'm still working on the PR that allows to use different strings for the same tag. I think that PR _may_ get past this issue.

Reply:
> I am assuming rrtmgp or other process relies of files with dimension name lw_band/sw_band
I don't think this is true actually.